### DISCUSSION TITLE
Deadline and test question

### DISCUSSION THREAD

**Discussion by superma-C** [geeksblabla/blanat#110] [Q&A] [answered]
Hello, I just learned about this competition, when is the deadline please?
Also can you please confirm that the gen.py file is the same used to generate the test set? if not are there any changes in the list of cities or fruits?
Thank you

**Comment by aboullaite** [comment, marked as answer]
All those questions are answered in the readme :) and yes, we used the same script to generate the final data

**Reply by superma-C** [reply]
I thought the winner will be announced during the conf, thank you :)
